tests/Feature/FlightControllerTest.php

```diff
@@ -2,547 +2,570 @@
 
 namespace Tests\Feature;
 
-use App\Http\Controllers\FlightController;
+use App\Actions\Flight\UpdateFlightAction;
+use App\Http\Controllers\Flight\DeleteFlightController;
+use App\Http\Controllers\Flight\GetFlightController;
+use App\Http\Controllers\Flight\StoreFlightController;
+use App\Http\Controllers\Flight\UpdateFlightController;
 use App\Models\Airline;
 use App\Models\City;
 use App\Models\Flight;
 use Illuminate\Foundation\Testing\RefreshDatabase;
 use Tests\TestCase;
 
-class FlightControllerTest extends TestCase
-{
-    use RefreshDatabase;
-
-    /** @test */
-    public function testGetFlightsForAnAirline()
-    {
-        City::factory(2)->create();
-
-        Airline::factory()->create();
-
-        Flight::create([
-            'airline_id' => 1,
-            'origin_city_id' => 1,
-            'destination_city_id' => 2,
+it('gets flights for an airline ', function () {
+    $origin = City::factory()->create();
+    $destination = City::factory()->create();
+
+    $airline = Airline::factory()->create();
+
+    Flight::create([
+        'airline_id' => 1,
+        'origin_city_id' => 1,
+        'destination_city_id' => 2,
+        'departure_time' => '2023-05-05T14:30:00',
+        'arrival_time' => '2023-05-05T14:31:00',
+    ]);
+
+    $this->json('GET', action(GetFlightController::class))
```

Comment:
instead of  `action(GetFlightController::class)`, can we do `route('flights.get')` or whatever the name of the function may be (you may need to add names to the routes in the api file).
that way we test the full Flight enpoints